Okta/OktaAuth/Tasks/OktaUserInfo.swift

```diff
@@ -30,18 +36,18 @@ internal struct UserInfo {
             "Authorization": "Bearer \(token)"
         ]
 
-        OktaApi.post(userInfoEndpoint, headers: headers, postData: nil,
+        oktaAPI.post(userInfoEndpoint, headers: headers, postData: nil,
             onSuccess: { response in callback(response, nil)},
```

Comment:
As a follow-up, we need to validate the `/userinfo` response [correctly](https://openid.net/specs/openid-connect-core-1_0.html#UserInfoResponse).

The approach we've taken elsewhere, is to modify the `getUser` method to do the following:
1. If an `accessToken` and `idToken` are present, request additional information from the `/userinfo` endpoint (today's implementation)
2. If an `idToken` is available but there is no valid `accessToken` (doesn't exist or is expired) - Decode the `idToken` JWT and return the user's claims
3. If an `accessToken` is available but there is no valid `idToken` (doesn't exist or is expired) - Fail due to not being able to validate the userinfo API response
4. If no `accessToken` or `idToken` is available, fail.

Reply:
@anastasiiaiurok-okta , I will create jira ticket for this enhancement. Suggest to implement it in next sprint